CITATION.cff

```diff
@@ -0,0 +1,81 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: Frictionless Python Framework
+message: >-
+  To cite the Frictionless Python Framework in publications
+  please use:
+type: software
+authors:
+  - given-names: Evgeny
+    family-names: Karev
+    email: eskarev@gmail.com
```

Comment:
Affiliation `Open Knowledge Foundation`?

Reply:
@roll do you want to be listed as affiliated to OKFN or you'd rather not?

Reply:
@peterdesmet 
Does it mean historical affiliation?

It's historical then OKFN if current Datist

Reply:
You'd say current one. You can update it when it changes or leave blank. Since you're the first author, I would indicate it though.